manifests/client.pp

```diff
@@ -51,27 +51,27 @@
 #   class { 'bacula::client': director_name => 'mydirector.example.com' }
 #
 class bacula::client (
-  Array[String]           $packages,
-  String                  $services,
-  String                  $default_pool,
-  Optional[String]        $default_pool_full,
-  Optional[String]        $default_pool_inc,
-  Optional[String]        $default_pool_diff,
-  Integer                 $port                = 9102,
-  Array[String[1]]        $listen_address      = [],
-  String                  $password            = 'secret',
-  Integer                 $max_concurrent_jobs = 2,
-  String                  $director_name       = $bacula::director_name,
-  Bacula::Yesno           $autoprune           = true,
-  Bacula::Time            $file_retention      = '45 days',
-  Bacula::Time            $job_retention       = '6 months',
-  String                  $client              = $trusted['certname'],
-  String                  $address             = $facts['networking']['fqdn'],
-  Optional[Bacula::Yesno] $pki_signatures      = undef,
-  Optional[Bacula::Yesno] $pki_encryption      = undef,
-  Optional[String]        $pki_keypair         = undef,
-  Optional[String]        $pki_master_key      = undef,
-  Optional[String]        $plugin_dir          = undef,
+  Array[String[1]]               $packages,
```

Comment:
I think all these spaces in class parameters make things worse. In the worst case, when a paremeter is added or removed, all other parameters need to be aligned again. Sure, these spaces slightly improve readability, but I don't think it's worth it (and it is not covered by puppet-lint).

This PR would be a great opportunity to drop all spaces in class parameters.

Reply:
I actually prefer the align variable name and align value style. And the re-indentation on change is not a big issue IMHO. I've seen this discussion before. I think we should have a voxpupuli styleguide (and linter) that finally defines the desired layout. Maybe something to discuss on the ML?

Reply:
This was discussed a few times with no consensus as some people *can bear it* and other people *need it* to work efficiently, and the status quo is basically "just let people who work with a module do as they like as long as each commit does not switch from one style to another": https://github.com/voxpupuli/voxpupuli.github.io/pull/322

Anyway, be aware that there is an option to not show white-space changes on GitHub (the UI is crappy as hell to find it, the option is on the diff tab, before the actual diff).  This makes such annoyances much lower when they happen.